### ISSUE TITLE
Make Arcade use the new SignToolTask that accept a list of containers

### ISSUE THREAD

**Issue by JohnTortugo** [dotnet/arcade#464]
Once this PR #410 is merged we need to patch Arcade to invoke the SignToolTask passing a list of containers (instead of the path to SignToolData.json) to be "signed".

Cc: @maririos @tmat @weshaggard @jaredpar

**Comment by JohnTortugo** [comment]
How should we create the list of packages that will be passed by to the SignTool to sign?

I see that the current publishing logic get's the package directly [from a directory](https://github.com/dotnet/arcade/blob/d3a73e5629eced2995d0b805397953bdc6874b48/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L45). Can/should we just implement the same?

**Comment by tmat** [comment]
`Include="$(ArtifactsPackagesDir)**\*.nupkg"`
`Include="$(VisualStudioSetupOutputPath)**\*.vsix"`

**Comment by JohnTortugo** [comment]
Which other default Certificate/Strong-Name pairs we will need to set? So far I've only the one that is in our current SignToolData.json:

```
    <ItemGroup>
      <StrongNameSignInfo Include="MsSharedLib72" PublicKeyToken="31bf3856ad364e35" CertificateName="MicrosoftSHA2" />
    </ItemGroup>

```

**Comment by weshaggard** [comment]
See https://github.com/dotnet/arcade/issues/272.

We need to have default configuration for at least the list of strong names in those 2 links in the issue.

**Comment by weshaggard** [comment]
Reopening until the arcade repo itself is consuming these changes, unless you have another issue for that.
